pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -196,6 +196,14 @@ message LinuxSandboxSecurityContext {
     bool privileged = 6;
 }
 
+// Sysctls holds linux sysctls config for the sandbox.
+message Sysctls {
+    // List of safe sysctls which are set for the sandbox.
+    map<string, string> sysctls = 1;
+    // List of unsafe sysctls which are set for the sandbox.
+    map<string, string> unsafe_sysctls = 2;
```

Comment:
I am not sure what sysctls are namespaced, but it seems the namespaces of the ones we support (mostly .net) are shared among containers in the pod. In the future, if that's not the case, we'll have to make sure they are set for individual containers as well. Since unsafe ones are only experimental, I think it's okay for now.